vllm/v1/executor/multiproc_executor.py

```diff
@@ -234,6 +289,22 @@ def take_draft_token_ids(self) -> DraftTokenIds | None:
             "take_draft_token_ids", unique_reply_rank=self.output_rank
         )
 
+    def get_response_mqs(
```

Comment:
suggest rename to indicate private
```suggestion
    def _get_response_mqs(
```